domains/brodes.json

```
@@ -0,0 +1,12 @@
+{
+  "description": "small Website of mine, may have done it wrong",
+  "repo": "https://github.com/Br0des/Website",
+  "owner": {
+    "username": "br0des",
+    "email": "brodyyeah@proton.me"
+  },
+  "record": {
+    "CNAME": "https://br0des.github.io/Website/index.html"
```

```suggestion
    "CNAME": "br0des.github.io"
```
CNAME records cannot contain protocols (`https://`) or slashes (`/`).